src/quantum/config.js

```diff
@@ -7,48 +7,149 @@ import { getSignatures } from "../vendor/perfherder";
 
 const DEBUG = false;
 
-const CONFIG = {
-  windows64Regression: [
+const BENCHMARKS =
+  [
     {
       title: 'MotionMark HTML suite',
-      secondLink:
-        '/quantum/10/windows10-64/raptor-motionmark-htmlsuite-firefox/pgo',
-      signatures: {
-        'windows10-64': '8f014459793e2e94c3244d5edeaada0452b0c627',
-      },
-      repo: 'mozilla-central',
-      framework: 10,
+      label: "Firefox",
+      bits: 64,
+      filter: { and: [
+        { missing: "test"},
+        { eq: {
+          suite: "raptor-motionmark-htmlsuite-firefox",
+          repo: "mozilla-central",
+          framework: 10,
+          platform: "windows10-64"
+        }}
+      ]}
```

Comment:
@davehunt  Please use the [Files Changed](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/552/files) view to see the code above my comment in full

Above, is the replacement config for MotionMark HTML suite